packages/extension-ui/src/Popup/Signing/Details.tsx

```diff
@@ -30,7 +30,7 @@ function renderMethod (data: string, meta?: Metadata | null) {
     );
   }
 
-  const method = new Method(data, { meta });
+  const method = new GenericCall(data, { metadata: meta });
```

Comment:
So not sure about this one at all - so the idea with the API is that if this happens, that it should be the full metadata. And then... in steps the extension... it doesn't have any API instance (only types) and maybe it has updated metadata available.

Reply:
Not sure I understand; `meta` is the full metdata. Currently the full metadata is hard-coded in the extension ui, and passed to the Method/GenericCall constructor. Do we want to change this?